src/Akeneo/AkeneoClient.cs

```diff
@@ -62,16 +62,32 @@ public AkeneoClient(Uri apiEndPoint, IAuthenticationClient authClient) : base(ap
 			return FilterAsync<TModel>(queryString, ct);
 		}
 
-		public async Task<PaginationResult<TModel>> FilterAsync<TModel>(string queryString, CancellationToken ct = default(CancellationToken)) where TModel : ModelBase
+        public Task<PaginationResult<TModel>> SearchAsync<TModel>(IEnumerable<Criteria> criterias, int limit = 10, CancellationToken ct = default(CancellationToken)) where TModel : ModelBase
```

Comment:
I think it makes sense with a limit parameter here. But instead of creating a new method, we should extend the existing one and have a check like

```csharp
if(limit > 0)
{
  // append limit to query string
}
```